pkg/client/client_ops.go

```diff
@@ -47,7 +47,7 @@ func (c *OpClient) ParseOps(
 	ops = append(ops, feeOps...)
 
 	ops = append(ops, handlers.MintOps(tx, len(ops))...)
-	ops = append(ops, handlers.BurnOps(tx, len(ops))...)
+	// ops = append(ops, handlers.BurnOps(tx, len(ops))...)
```

Comment:
Can you explain what kind of block API issue we hit? And why need to comment out burn ops parser to fix it?

Reply:
Already discussed this offline